wp_api/src/api_error.rs

```diff
@@ -15,6 +15,8 @@ pub enum RequestExecutionError {
 
 #[derive(Debug, PartialEq, Eq, thiserror::Error, uniffi::Error)]
 pub enum WpApiError {
+    #[error("Status code ({}) is not valid", status_code)]
+    InvalidStatusCode { status_code: u16 },
```

Comment:
Nitpick: This error is for HTTP status code like `1000`, right? What do you think making it more explicit, like `InvalidHTTPStatusCode`?

Reply:
Sure - I'll make that change in a subsequent PR since this got auto-merged. I think it needs to be `InvalidHttpStatusCode` though to follow naming guidelines.